### ISSUE TITLE
Training set and seed

### ISSUE THREAD

**Issue by jungaloo** [lululxvi/deepxde#353]
Hi,

I'm playing around with Poisson_Lshape.py and I was wondering where and how is computed the training set?
Also, I'm trying to set the seed, the way the points are scattered around in the geometry, so I can try and compute a L2 error or at least reproduce the results, but I'm not sure where to look.

Thank you for the help.

**Comment by lululxvi** [comment]
The training data will be saved in the file train.dat

You should set the seed of both numpy and tensorflow. If you really want to control the sampled points, you can set `num_domain` and `num_boundary` to be 0, so that DeepXDE won't sample any points. Then you sample the points by your self, and pass it to `anchors`, so that DeepXDE will only use these points.

**Comment by lululxvi** [comment]
Set random seed has been supported by `deepxde.config.set_random_seed()`.
